### PR TITLE
[WIP]Adding managed-cluster-config services

### PR DESCRIPTION
**Pull request by rafael-azevedo**
This PR adds in the service yaml required to push the manged-cluster-config selectorsyncset

### CODE REVIEW

File: managed-cluster-config-services/manged-cluster-config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  name: managed-cluster-config
+  path: 00-osd-manged-cluster-config.selectorsyncset.yaml 
+  url: https://github.com/openshift/managed-cluster-config 
+  hash_length: 7
+  environments:
+  - name: production
+    parameters:
+  - name: staging
+    parameters:
+
```
Comment (jharrington22) [inline managed-cluster-config-services/manged-cluster-config.yaml]: Remove the blank lines?

File: managed-cluster-config-services/manged-cluster-config.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: 00-osd-manged-cluster-config.selectorsyncset.yaml 
+  url: https://github.com/openshift/managed-cluster-config 
+  hash_length: 7
+  environments:
+  - name: production
+    parameters:
```
Comment (jharrington22) [inline managed-cluster-config-services/manged-cluster-config.yaml]: @jmelis should this be blank or removed completely?

File: managed-cluster-config-services/manged-cluster-config.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: 00-osd-manged-cluster-config.selectorsyncset.yaml
```
Comment (jharrington22) [inline managed-cluster-config-services/manged-cluster-config.yaml]: @rafael-azevedo the original path is prefixed with a `/` so I would think that is also needed here cc/ @jmelis

**Comment by jharrington22** [comment]
@rafael-azevedo we Also need to update the context here in https://github.com/app-sre/saas-hive/blob/master/config.yaml

File: managed-cluster-config-services/manged-cluster-config.yaml (hunk 3 above)
Comment (jmelis) [reply managed-cluster-config-services/manged-cluster-config.yaml]: :+1:

File: managed-cluster-config-services/manged-cluster-config.yaml (hunk 2 above)
Comment (jmelis) [reply managed-cluster-config-services/manged-cluster-config.yaml]: you can actually remove the whole `environments` key

**Comment by xyntrix** [comment]
this looks ok to me. ok to rebase? master has some updates that need to go in here.

**Pull request closed without merging**